docs/helm-broker/03-03-create-addons-repo.md

```diff
@@ -112,7 +112,104 @@ spec:
     - url: "bitbucket.org/kyma-project/addons//addons/index.yaml"
 ```
 
->**NOTE:** For now, the SSH protocol is not supported.
-
   </details>
 </div>
+
+## Authorization
+
+The AddonsConfiguration and ClusterAddonsConfiguration custom resources allows you to define authorization as a part of the URL. For more details please read the [go-getter protocols](https://github.com/hashicorp/go-getter/blob/master/README.md#general-all-protocols) description.
+Using sensitive information, like a password, directly in the URL is a not good approach. This can be solved by putting such data in a Secret resource and take advantage of a templating. You can use placeholders which refers to keys in the Secret. For example:
```

Comment:
```suggestion
Using sensitive information, such as passwords, directly in the URL is a not a good approach. This can be solved by putting such data in a Secret resource and taking advantage of templating. You can use placeholders which refer to keys in the Secret. For example:
```